lib/Utils.js

```diff
@@ -49,14 +49,15 @@ function unserializeUids(stringdata) {
 */
 function objectCopy(target) {
     const result = target;
-    let source;
-    function copy(f) {
-        result[f] = source[f];
-    }
     /* eslint-disable prefer-rest-params */
-    for (let i = 1; i < arguments.length; i++) {
-        source = arguments[i];
-        Object.keys(source).forEach(copy);
+    const nb = arguments.length;
+    for (let i = 1; i < nb; i++) {
+        const source = arguments[i];
+        const keys = Object.keys(source);
+        const keysNb = keys.length;
+        for (let j = 0; j < keysNb; j++) {
```

Comment:
Thanks for the sample code. I would not normally agree with micro-optimizations as they trade readability for intangible performance gain. Werelogs though, since it's used across all projects and that a request path can contain 50 more logs with objects which will be copied many times, the problem gets compounded. So, I am ok with this optimization.